openweather/src/main/java/action/SearchAction.java

```diff
@@ -0,0 +1,24 @@
+package action;
+
+import pages.HomePage;
+import pages.SearchPage;
+
+import java.util.List;
+
+public class SearchAction extends ActionBase {
+
+    private HomePage homePage;
+    private SearchPage searchPage;
+
+    public SearchAction() {
+        super();
+
+        homePage = getPageInstance(HomePage.class);
+        searchPage = getPageInstance(SearchPage.class);
```

Comment:
code hay qua !!!

Reply:
ờ, hay

Reply:
should rename to ActionSearch for consistency with ActionBase???